src/components/ReviewById.jsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsOfReview, getSingleReview, patchVotes } from "../api";
+import PostComment from "./PostComment";
+
+export const ReviewById = () => {

```

Comment:
Cant navigate back to home page without changing URL or pressing back, could we have a link somewhere for the home page. This would probably be better suited in another component sauch as a header or nav bar